README.md

```diff
@@ -29,4 +29,24 @@ model.compile(optimizer="sgd",
               metrics=[keras_metrics.precision(), keras_metrics.recall()])
 ```
 
+Similar configuration for multi-label binary crossentropy:
+```py
+import keras
+import keras_metrics
+
+model = models.Sequential()
+model.add(keras.layers.Dense(1, activation="sigmoid", input_dim=2))
+model.add(keras.layers.Dense(2, activation="softmax"))
+
+# Calculate precision for the second label.
+precision = keras_metrics.precision(label=1)
+
+# Calculate recall for the first label.
+recall = keras_metrics.precision(label=0)
+
```

Comment:
We should enable(limit indeed) users to use precision and recall without defining label
```
model = models.Sequential()
model.add(keras.layers.Dense(10, activation="sigmoid", input_dim=2))
model.add(keras.layers.Dense(2, activation="softmax"))
precision = keras_metrics.precision()
recall = keras_metrics.precision()
```
For the multi-classification, on the other hand layer should be indicated
```
model2 = models.Sequential()
model2.add(keras.layers.Dense(10, activation="sigmoid", input_dim=2))
model2.add(keras.layers.Dense(3, activation="softmax"))
precision2 = keras_metrics.precision(label=1)
recall2 = keras_metrics.precision(label=1)
```

Reply:
Parameter ```label``` is optional, if it's not specified, metric works like before, so the changes are backward-compatible.

Reply:
That's what exactly I meant indeed if someone puts two units without labeling, the answer will be wrong(because layer has to be equal to 1 for two units). But it is matter of coding actually and I am not an expert to be honest. Just go over the options for you to decide
1-) Merged as it is and give the responsibility to the user.
2-) For one unit and two units, it is definitely clear which layer to choose. And 99% of the people who use this library will be having one or two layers. So you can offer them a library that choose the correct layer to calculate precision and recall for one unit and for two units.